README.md

```diff
@@ -23,9 +23,13 @@ skip that if you want to run it from the repl.
 
 Kaocha runner exposes four commands:
```

Comment:
There are now five commands. Maybe we should stop counting them here. :)

Reply:
whoops. missed this. will fix.